src/components/CurriculumComponents/UnitsTab/UnitsTab.tsx

```diff
@@ -214,11 +231,21 @@ const UnitsTab: FC<UnitsTabProps> = ({ data, examboardSlug }) => {
         });
       }
       filters.tiers.sort((a, b) => a.tier_slug.localeCompare(b.tier_slug));
+      // Sort disciplines
+      filters.disciplines.sort((a, b) => a.title.localeCompare(b.title));
+
+      // Add an "All" option if there are 2 or more disciplines. Set to -1 id as this shouldn't ever appear in the DB
+      const allDisciplineTag: Discipline = { id: -1, title: "All" };
```

Comment:
Why is it greater than 2 and not 1? Will there be a default of 2 if there are 2, or are there no single disciplines

Reply:
Greater than or equal to 2. So, my theory was that there is no point showing the filter unless there are at least 2 different disciplines to choose from.

Reply:
Ah, got you. Makes sense 👍